content/docs/ops/service-disruption-guide.md

```diff
@@ -0,0 +1,58 @@
+---
+menu:
+  docs:
+    parent: policies
+title: Service disruption guide
+---
+
+This guide outlines how we handle communications for service disruptions during East Coast business hours.
+
+Not a guide for:
+
+* Hours outside of East Coast business hours -- out of scope for this document.
+* Security incidents -- switch to [security incident guide]({{< relref "docs/ops/security-ir.md" >}}).
+* Major service disruptions (more than 30 minutes of unexpected downtime or significantly reduced service) -- switch to [contingency plan]({{< relref "docs/ops/contingency-plan.md" >}}).
```

Comment:
I agree that we should take another look at the contingency plan. That should probably be its own story, since I'd want to check for compliance effects - we may need to check with our FedRAMP team when making significant changes to it.